RELEASING.md

```diff
@@ -36,6 +36,16 @@ The file will be located under ./artemis-pom/RELEASE/
 Remove these files manually under Nexus while the repository is still open.
 
 
+## Checking out a new empty git repository
+
+Before starting make sure you clone a brand new empty the git repository as follows as the release plugin will use the upstream for pushing the tags:
+
+```sh
+git clone git://github.com/apache/activemq-artemis.git
+cd activemq-artemis
+git remote add upstream https://git-wip-us.apache.org/repos/asf/activemq-artemis.git
```

Comment:
Yep, then you can push wherever you want.  Usually we only push tags after the release is fully executed.  So its ok if its sitting out on your mirror.